bin/respirate

```diff
@@ -6,29 +6,20 @@ require_relative "../loader"
 d = Scheduling::Dispatcher.new
 Signal.trap("TERM") { d.shutdown }
 
-if Config.heartbeat_url
-  puts "Starting heartbeat prog"
-  # We always insert the heartbeat using the same UBID ("stheartbeatheartbheartheaz")
-  Strand.dataset.insert_conflict.insert(id: "8b958d2d-cad4-5f3a-5634-b8b958d45caf", prog: "Heartbeat", label: "wait")
+workers = [
+  ["stresolvee4block0dnsnamesz", "ResolveGloballyBlockedDnsnames"],
+  ["stcheckzvsagezalertszzzzza", "CheckUsageAlerts"],
+  ["stexp1repr0ject1nv1tat10na", "ExpireProjectInvitations"],
+  ["st10g0vmh0st0vt111zat10nzz", "LogVmHostUtilizations"],
+  Config.heartbeat_url ? ["stheartbeatheartbheartheaz", "Heartbeat"] : nil,
+  Config.github_app_id ? ["stredelivergith0bfail0reaz", "RedeliverGithubFailures", [{last_check_at: Time.now}]] : nil
+]
+workers.compact!
+
+workers.each do |ubid, prog, stack|
+  Strand.dataset.insert_conflict.insert(id: UBID.parse(ubid).to_uuid, prog:, label: "wait", stack: (stack || [{}]).to_json)
```

Comment:
I would use `import` instead of `insert`, and then you can handle everything in one `INSERT` statement.

Reply:
How do we deal with conflicts when using `import`?

Reply:
I changed it